api/envoy/extensions/transport_sockets/s2a/v3alpha/s2a.proto

```diff
@@ -0,0 +1,23 @@
+syntax = "proto3";
+
+package envoy.extensions.transport_sockets.s2a.v3alpha;
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.transport_sockets.s2a.v3alpha";
+option java_outer_classname = "S2aProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).work_in_progress = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Secure Session Agent (S2A)]
+// [#extension: envoy.transport_sockets.s2a]
```

Comment:
it might be that this is causing the fail - perhaps removing it until the source code lands would work - not sure

Reply:
Yep, this is why. I would add a not implemented annotation instead
` // [#not-implemented-hide:]`

Reply:
Update the code to remove #extension: envoy.transport_sockets.s2a comment.